btrack/_localization.py

```diff
@@ -34,7 +34,7 @@
     DASK_INSTALLED = False
 
 # get the logger instance
-logger = logging.getLogger("worker_process")
+logger = logging.getLogger("__name__")
```

Comment:
I think this needs to be `__name__` rather than `"__name__"`

```suggestion
logger = logging.getLogger(__name__)
```

Reply:
D'oh, should be fixed now.